src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorSupervisor.scala

```diff
@@ -116,7 +126,13 @@ class ClusterMonitorSupervisor(monitorConfig: MonitorConfig, dataprocConfig: Dat
       startClusterMonitorActor(cluster)
 
     case AutoFreezeClusters =>
-      autoFreezeClusters
+      autoFreezeClusters()
+
+    case Tick =>
```

Comment:
Again I'd like this message to be more descriptive than `Tick` -- maybe `CheckForClusters`